src/phpDocumentor/DocumentationRepository.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * phpDocumentor
+ *
+ * PHP Version 5.5
+ *
+ * @copyright 2010-2015 Mike van Riel / Naenius (http://www.naenius.com)
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor;
+use phpDocumentor\Project\VersionNumber;
+use Stash\Pool;
```

Comment:
You are absolutely right; I just re-read that section of the docs and the Pool item is more appropriate